e2e/tests/ui-driven/src/helpers/geospatialChecks.ts

```diff
@@ -51,10 +51,22 @@ export const checkGeoJsonContent = async (
   // Get the geojsonData attribute
   const geojsonData = await mapComponent.getAttribute(attribute);
 
-  expect(
-    JSON.parse(geojsonData!),
-    "map attribute matches expected mock attribute",
-  ).toEqual(geoJson);
+  expect(geojsonData).toBeDefined();
+
+  // Exact GeoJSON coords can change with version bumps of OpenLayers and Playwright
+  // Compare using .toBeCloseTo() assertion
+  const coords: Array<[number, number]> = JSON.parse(geojsonData!).geometry
+    .coordinates[0];
+  const expectedCoords = geoJson.geometry.coordinates[0];
+
+  coords
+    .flat(2)
+    .forEach((coord, index) =>
+      expect(
+        coord,
+        "map attribute matches expected mock attribute",
+      ).toBeCloseTo(expectedCoords.flat(2)[index]),
+    );
```

Comment:
This was failing as the two GeoJSONs didn't match. A few reasons this could happen - 
 - Bumps to map repo (OL version, proj4 version)
 - Bumps to Playwright

Something in the mapping of mouse x & y to coords gives an inexact / unreliable response - these coordinates are always very close but never quite exact.

Relevant XKCD - https://xkcd.com/2170/